Drop-In/src/main/res/layout/bt_fragment_supported_payment_methods.xml

```diff
@@ -11,85 +11,116 @@
-                    android:layout_alignParentStart="true"
-                    android:layout_marginStart="20dp"
-                    android:gravity="center_vertical"
-                    android:text="@string/bt_recent" />
+                    android:paddingLeft="4dp"
+                    android:paddingRight="4dp" />
 
-            </RelativeLayout>
+                <include layout="@layout/bt_section_divider" />
+
+            </LinearLayout>
+
+            <TextView
+                android:id="@+id/bt_supported_payment_methods_header"
+                style="@style/bt_choose_payment_label"
+                android:paddingTop="@dimen/bt_choose_new_payment_label_padding_top"
+                android:paddingBottom="@dimen/bt_payment_label_padding_bottom"
+                android:text="@string/bt_other"
+                android:visibility="gone"
+                />
 
             <androidx.recyclerview.widget.RecyclerView
-                android:id="@+id/bt_vaulted_payment_methods"
+                android:id="@+id/bt_supported_payment_methods"
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
                 android:paddingLeft="4dp"
                 android:paddingRight="4dp" />
 
-            <include layout="@layout/bt_section_divider" />
-
         </LinearLayout>
 
-        <TextView
-            android:id="@+id/bt_supported_payment_methods_header"
-            style="@style/bt_choose_payment_label"
-            android:paddingTop="@dimen/bt_choose_new_payment_label_padding_top"
-            android:paddingBottom="@dimen/bt_payment_label_padding_bottom"
-            android:text="@string/bt_select_payment_method" />
-
-        <androidx.recyclerview.widget.RecyclerView
-            android:id="@+id/bt_supported_payment_methods"
+        <RelativeLayout
+            android:id="@+id/bt_select_payment_method_loader_wrapper"
             android:layout_width="match_parent"
-            android:layout_height="wrap_content"
-            android:paddingLeft="4dp"
-            android:paddingRight="4dp" />
-
-    </LinearLayout>
-
-    <LinearLayout
-        android:id="@+id/bt_select_payment_method_loader_wrapper"
-        android:layout_width="match_parent"
-        android:layout_height="match_parent"
-        android:background="@color/bt_base_background"
-        android:gravity="center"
-        android:orientation="vertical">
-
-        <ProgressBar
-            android:id="@+id/bt_select_payment_method_loader"
-            android:layout_width="@dimen/bt_progress_bar_diameter"
-            android:layout_height="@dimen/bt_progress_bar_diameter"
-            android:layout_gravity="center"
-            android:indeterminate="true" />
+            android:layout_height="match_parent"
+            android:background="@color/bt_base_background"
+            android:orientation="vertical">
+
+            <ProgressBar
```

Comment:
For some reason - moving this progress bar and it's wrapper within the linear layout that includes the cancel button and select payment method header messes up the centering. Now the progress bar is centered vertically, but not horizontally. Keeping this PR in draft until we can figure out the centering.